src/app/Directives/email.validator.ts

```diff
@@ -0,0 +1,26 @@
+import {Directive} from '@angular/core';
```

Comment:
Since you already have `validateEmail` function in `global.service.ts`, no need to create the same function again here. (Also remove the log statement)

Reply:
This directive uses validateEmail function in globalService itself.
Done